docs/storefront/graphql/examples/index.mdx

```diff
@@ -208,4 +208,18 @@ query {
 }
 ```
 
-<a href="https://developer.bigcommerce.com/graphql?playground_tab=popularBrands" target="_blank">**Try it in GraphQL Playground**</a>
+<a href="https://developer.bigcommerce.com/graphql-storefront/playground?tabs=popularBrands" target="_blank">**Try it in GraphQL Playground**</a>
+
+## Add a favicon to Storefront GraphQL
```

Comment:
It's get favicon. WE use `query` for read operations and `mutation` for anything that change data